actionpack/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+*   Add `-g` and `-c` options for `rake routes` task. This options returns the url `name`,
+    `verb`, `path` and `reqs` field that match the pattern or returns routers for specific
```

Comment:
It would be nice to mention that `-g` is short for grep and `-c` for controller.